prisma/migrations/00_init/migration.sql

```diff
@@ -0,0 +1,489 @@
+-- CreateTable
```

Comment:
Are these 3 migrations auto generated or manually put together from a schema dump? 

I wonder what the leader 0 is good for :thinking: I hope it can handle > 99 migrations :sweat_smile:

Reply:
Yes, the first `00_init` migration is generated from the schema and the rest are just manually dumped from the prod database. The `0X_`-convention was just something I picked, but prisma would then use a date prefix instead, e.g. `202304301430....`. The migrations are run in order of file names, so I just wanted to use the leading `0` to make sure they are run first.